modules/UI/util/MessageHandler.js

```diff
@@ -374,7 +337,7 @@ const messageHandler = {
             box: '',
             form: '',
             prompt: `dialog aui-layer aui-dialog2 aui-dialog2-${size}`,
-            close: 'aui-icon aui-icon-small aui-iconfont-close-dialog',
+            close: 'aui-hide',
```

Comment:
I'm assuming you purposefully hid the X for consistency with AtlasKit icons.

Reply:
Correct.